docs/_includes/arguments/date-picker.md

```diff
@@ -0,0 +1,16 @@
+| Name         | Type   | Required | Description                                                                                                                      |
+| ------------ | ------ | -------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| id           | string | Yes      | The ID of the input.                |
+| name         | string | Yes      | The name of the input, which is submitted with the form data. |
+| value        | string | No       | Optional initial value of the input. |
+| formGroup    | object | No       | Additional options for the form group containing the text input component. See [formGroup](#options-date-picker-form-group). |
+| label        | object | Yes      | The label used by the text input component. See [GOV.UK text input documentation](https://design-system.service.gov.uk/components/text-input/) for label options. |
+| hint         | object | No       | Can be used to add a hint to a text input component. See [GOV.UK text input documentation](https://design-system.service.gov.uk/components/text-input/) for hint options. |
+| errorMessage | object | No       | Can be used to add an error message to the text input component. The error message component will not display if you use a falsy value for `errorMessage`, for example `false` or `null`. See [GOV.UK text input documentation](https://design-system.service.gov.uk/components/text-input/) for errorMessage options. |
+| minDate      | string | No       | Earliest date that can be selected (format dd/mm/yyyy) |
+| maxDate      | string | No       | Latest date that can be selected (format dd/mm/yyyy) |
+| exludedDates | string | No       | String of pace separated dates that cannot be selected |
+| excludedDays | string | No       | String of space separated days of the week that cannot be selected |
+| weekStartDay | string | No       | Day of the week the calendar starts on. Either 'monday' or 'sunday'. Defaults to 'monday' |
```

Comment:
Interesting point.  I haven't really changed this, just gone with how it was.  My suspicion is that it was space separated as that is how it would have to be in a data-attribute.

I suppose we could pass in an array to the nunjucks component, and translate that into an array.  We could also then allow an array to be passed in via the JS api, however this would then introduce a complication in the code where data-attributes would be supplied as a string, and JS options as an array.

I'm not 100% on this, but I feel that GDS tends to only use nunjucks arrays to pass in lists of objects (e.g. items in a task list), and their JS API schema only allows strings, numbers, booleans and objects.

I wonder if the thinking is that a space-separated string is slightly easier for non-developers to use in prototypes etc...